shell/common/shell_test_external_view_embedder.cc

```diff
@@ -61,6 +65,12 @@ void ShellTestExternalViewEmbedder::SubmitFrame(
     GrDirectContext* context,
     std::unique_ptr<SurfaceFrame> frame) {
   frame->Submit();
+  if (frame && frame->SkiaSurface()) {
```

Comment:
Windows toolchain doesn't seem to be liking this:
```
C:\b\s\w\ir\kitchen-checkout\depot_tools\win_toolchain\vs_files\418b3076791776573a815eb298c8aa590307af63\win_sdk\bin\..\..\VC\Tools\MSVC\14.16.27023\include\atomic(620,2): error: static_assert failed due to requirement 'alignof(SkISize) >= sizeof(unsigned long long)' "You've instantiated std::atomic<T> with sizeof(T) equal to 2/4/8 and alignof(T) < sizeof(T). Before VS 2015 Update 2, this would have misbehaved at runtime. VS 2015 Update 2 was fixed to handle this correctly, but the fix inherently changes layout and breaks binary compatibility. Please define _ENABLE_ATOMIC_ALIGNMENT_FIX to acknowledge that you understand this, and that everything you're linking has been compiled with VS 2015 Update 2 (or later)."
        static_assert(alignof(_Ty) >= sizeof(_My_int),
        ^             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:\b\s\w\ir\kitchen-checkout\depot_tools\win_toolchain\vs_files\418b3076791776573a815eb298c8aa590307af63\win_sdk\bin\..\..\VC\Tools\MSVC\14.16.27023\include\atomic(635,5): note: in instantiation of template class 'std::_Atomic_base<SkISize, 8>' requested here
                : _Atomic_base<_Ty, sizeof (_Ty)>
                  ^
../../flutter/shell/common/shell_test_external_view_embedder.h(86,24): note: in instantiation of template class 'std::atomic<SkISize>' requested here
  std::atomic<SkISize> last_submitted_frame_size_;
                       ^
1 error generated.
```


See https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8869006746431748192/+/steps/build_host_debug/0/stdout for full error.

Reply:
This isn't going to be an [issue in future](https://github.com/microsoft/STL/issues/717), in the meanwhile perhaps we could define `_ENABLE_ATOMIC_ALIGNMENT_FIX`. There's nothing wrong with the code, it's just a windows STL quirk.